packages/neon/neon/lib/src/widgets/linear_progress_indicator.dart

```diff
@@ -3,28 +3,32 @@ part of '../../neon.dart';
 class NeonLinearProgressIndicator extends StatelessWidget {
   const NeonLinearProgressIndicator({
     this.visible = true,
+    this.alwaysThere = true,
     this.margin = const EdgeInsets.symmetric(horizontal: 10),
     this.color,
     this.backgroundColor = Colors.transparent,
     super.key,
   });
 
   final bool visible;
+  final bool alwaysThere;
   final EdgeInsets? margin;
   final Color? color;
   final Color? backgroundColor;
 
   @override
-  Widget build(final BuildContext context) => Container(
-        margin: margin,
-        child: SizedBox(
-          height: 3,
-          child: visible
-              ? LinearProgressIndicator(
-                  color: color,
-                  backgroundColor: backgroundColor,
-                )
-              : null,
-        ),
-      );
+  Widget build(final BuildContext context) => !alwaysThere && !visible
+      ? Container()
```

Comment:
Instead of using a `Container()` as an empty widget a `SizedBox()` would be preferred as it has a constant constructor that could improve layout performance.